www/gatsby-config.js

```diff
@@ -75,6 +78,65 @@ module.exports = {
         trackingId: 'UA-41298772-1',
       },
     },
+    {
+      resolve: 'gatsby-plugin-feed',
+      options: {
+        query: `
+         {
+          site {
+            siteMetadata {
+              title: rssFeedTitle
+              description: rssFeedDescription
+              siteUrl
+              site_url: siteUrl
```

Comment:
Seems like we only need `site_url` in this query, if we update `site.siteMetadata.siteUrl` below to be `site.siteMetadata.site_url`?

Reply:
Ok sounds good!

Reply:
The problem is that for some reason the plugin requires that duplicate key to be present, otherwise it doesn't work correclty. So, I run this query:
```
{
          site {
            siteMetadata {
              title: rssFeedTitle
              description: rssFeedDescription
              site_url: siteUrl
            }
          }
        }
```

The XML I get back turns out like this:

```xml
<link>undefined/blog/2017/09/26/react-v16.0.html</link>
<guid isPermaLink="false">undefined/blog/2017/09/26/react-v16.0.html</guid>
```

Seems like even in the documentation page they suggest using the duplicate key:
https://www.gatsbyjs.org/packages/gatsby-plugin-feed/

Reply:
@KyleAMathews Why is this?

Reply:
I didn't write this plugin so… probably just should be fixed. I believe `site_url` is required to be the key name for the node.js package that creates the feed but the duplicate in the query doesn't seem necessary.

Reply:
Gotcha. Thanks for the context.